geode-core/src/distributedTest/java/org/apache/geode/distributed/LocatorUDPSecurityDUnitTest.java

```diff
@@ -41,28 +38,25 @@ protected void addDSProps(Properties p) {
   }
 
   @Test
-  public void testLocatorWithUDPSecurityButServer() throws Exception {
+  public void testLocatorWithUDPSecurityButServer() {
     disconnectAllFromDS();
-    VM vm0 = VM.getVM(0);
+    VM vm = VM.getVM(0);
 
     final int port = AvailablePort.getRandomAvailablePort(AvailablePort.SOCKET);
     DistributedTestUtils.deleteLocatorStateFile(port1);
     final String locators = NetworkUtils.getServerHostName() + "[" + port + "]";
 
-    vm0.invoke("Start locator " + locators, () -> startLocator(port));
-    try {
+    startLocatorWithSomeBasicProperties(vm, port);
 
-      Properties props = new Properties();
-      props.setProperty(MCAST_PORT, "0");
-      props.setProperty(LOCATORS, locators);
+    try {
+      Properties props = getBasicProperties(locators);
       props.setProperty(MEMBER_TIMEOUT, "1000");
-      // addDSProps(props);
-      system = (InternalDistributedSystem) DistributedSystem.connect(props);
+      system = getConnectedDistributedSystem(props);
```

Comment:
Does "getConnectedDistributedSystem" actually throw GemFireConfigException? If this test doesn't need that "catch (GemFireConfigException" then delete it. If this test is actually verifying that GemFireConfigException should be thrown then it needs some more help.

Either add a fail statement in just before the catch so the test fails if the expected exception is not thrown. Or rewrite the block to use catchThrowable:
```
import static org.assertj.core.api.Assertions.catchThrowable;

Throwable thrown = catchThrowable(() -> getConnectedDistributedSystem(props));
assertThat(thrown).isInstanceOf(GemFireConfigException.class).hasMessageContaining("Rejecting findCoordinatorRequest");
```
Lines like "vm.invoke(LocatorDUnitTest::stopLocator);" would typically be better in a tearDown() method rather than a finally block.

Reply:
Done. Added a fail statement.